Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM rocker/r-ver:4.0.0
 
 # install a newer-ish version of renv, but the specific version we want will be restored from the renv lockfile
-ENV RENV_VERSION 0.8.3-81
+ENV RENV_VERSION 0.13.2
 RUN R --quiet -e "source('https://install-github.me/rstudio/renv@${RENV_VERSION}')"
```

Comment:
this service I was using with older dockerfile and renv is going away soon. https://install-github.me/

let's use this instead:

```sh
RUN R --quiet -e "install.packages('remotes', repos = 'https://packagemanager.rstudio.com/all/__linux__/focal/latest')"
# make sure version matches what is used in the project: packageVersion('renv')
ENV RENV_VERSION 0.13.2
RUN R --quiet -e "remotes::install_github('rstudio/renv@${RENV_VERSION}')"
```

Reply:
oh, I didn't even notice that. will do!